get-deps.sh

```diff
@@ -3,18 +3,20 @@
 set -e
 
 if [ "$GOPATH" = "" ]; then
-    export GOPATH=$(mktemp -d)
+    GOPATH=$(mktemp -d)
+    export GOPATH
+    # shellcheck disable=SC2064
```

Comment:
This warning is https://github.com/koalaman/shellcheck/wiki/SC2064

What happens if we expand GOPATH later?

Reply:
Nothing exciting, we don't modify it elsewhere, so it'll effectively have the same value as it has here.